fdb-record-layer-icu/fdb-record-layer-icu.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ * fdb-record-layer-icu.gradle
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: rootProject.file('gradle/strict.gradle')
+
+def coreProject = ":${ext.coreProjectName}"
+dependencies {
+    compile project(coreProject)
+    compile "com.ibm.icu:icu4j:${icuVersion}"
+    compile "com.google.protobuf:protobuf-java:${protobufVersion}"
+    compile "org.slf4j:slf4j-api:${slf4jVersion}"
+    compileOnly "com.google.code.findbugs:jsr305:${jsr305Version}"
+    compileOnly "com.google.auto.service:auto-service:${autoServiceVersion}"
+
+    testCompile project(path: coreProject, configuration: 'tests')
+    testCompileOnly "com.google.code.findbugs:jsr305:${jsr305Version}"
+    testCompileOnly "com.google.auto.service:auto-service:${autoServiceVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-core:${log4jVersion}"
+
+    testImplementation "org.junit.jupiter:junit-jupiter-api:${junitVersion}"
+    testCompile "org.junit.jupiter:junit-jupiter-params:${junitVersion}"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${junitVersion}"
+}
+
+test {
+    useJUnitPlatform {
+        if (System.getenv('SKIP_FDB_TESTS') != null && System.getenv('SKIP_FDB_TESTS') == 'true') {
+            excludeTags 'RequiresFDB'
+        }
+    }
+}
```

Comment:
I believe this will need something to update the description to produce a POM file that MavenCentral will be happy with. The gradle files for the other subprojects now include something like it at the bottom for reference.

Reply:
I had a stab at it.